### PR TITLE
Updating the motivation part

### PR DESCRIPTION
**Pull request by bast**
Work in progress. Not to be merged yet but you can have a peek at what I am up to and comment.

### CODE REVIEW

**Comment by bast** [comment]
Ready for review. The getting started part will be done in a separate episode.

File: _episodes/01-motivation.md [hunk 1]
```diff
@@ -7,118 +7,108 @@ questions:
...
+```
 
 ---
 
-## Why Git?
+## Compare with Dropbox or Google Drive
+
+- Document/code is in one place, no need to email snapshots.
+- Possible to get old versions but in a much less useful way - snapshots of files, not directories.
+
```
Comment (rkdarst) [inline _episodes/01-motivation.md]: - How can you use an old version? (add to second point)
- What about if you want to work on multiple versions at the same time?  do you make a copy?  how do you merge copies?
- You rely on dropbox/google - what if you don't have internet?

File: _episodes/01-motivation.md [hunk 2]
```diff
@@ -7,118 +7,108 @@ questions:
...
+---
 
+## We will use [Git](https://git-scm.com) to record snapshots of our work - why Git?
 
-- Distributed
-- Other people use it
-- Merging and branching is easy and simple.
+- Easy to set up.
```
Comment (rkdarst) [inline _episodes/01-motivation.md]: Easy to set up - use even by yourself with no server needed.

File: _episodes/01-motivation.md [hunk 3]
```diff
@@ -7,118 +7,108 @@ questions:
...
+- Very popular: chances are high you will need to contribute to somebody else's code which is tracked with Git.
+- Distributed: good backup, no single point of failure, you can track and clean-up changes offline, simplifies collaboration model for open-source projects.
 - Important platforms such as [GitHub](https://github.com), [GitLab](https://gitlab.com), and [Bitbucket](https://bitbucket.org)
   build on top of Git.
 - Many platforms build on top of [GitHub](https://github.com).
 - Sharing software and data is getting popular and required in research context
   and [GitHub](https://github.com) is a popular platform for sharing software.
-- Can start using now, no need to configure a server 
-- *"Git is a four-handle, dual boiler espresso machine, not instant coffee."* [citation needed]
+- However, *"Git is a four-handle, dual boiler espresso machine, not instant coffee."* [citation needed]
```
Comment (rkdarst) [inline _episodes/01-motivation.md]: - git isn't the most user friendly and has its design quirks... but deep design is great and is definitely the most popular and what you are most likely to need to know.  So we teach it.

File: _episodes/01-motivation.md [hunk 4]
```diff
@@ -7,118 +7,108 @@ questions:
...
-    - implement code review
-    - decentralize access control
-    - decentralize bug exposure
-    - commit changes without network
-    - edit changes after they have been committed
+- Subversion is centralized (one server, many clients) and requires setting up and maintaining a server.
+- You cannot easily clean-up your recorded snapshots (commits) before you share them.
+- Not easy to get contributions from external contributors.
 
 ---
```
Comment (rkdarst) [inline _episodes/01-motivation.md]: Mercurial: many git concepts still apply.  For that matter, most important lesson is how and why to use version control, which applies to any system with some changes.

**Comment by bast** [comment]
Thank you for good comments! Another commit applying these is coming up.

**Pull request merged**